internal/pkg/deploy/cloudformation/pipeline_integration_test.go

```diff
@@ -40,12 +40,11 @@ func TestPipelineCreation(t *testing.T) {
 		createMockSecret(t, sm, secretId)
 		projCfClient := awsCF.New(projectSess)
 
-		const pipelineName = "pipepiper"
 		project := archer.Project{
 			Name:      randStringBytes(10),
 			AccountID: projectCallerInfo.Account,
 		}
-		pipelineStackName := project.Name + "-" + pipelineName
+		pipelineStackName := project.Name + "-pipepiper"
```

Comment:
wait, don't we need to get rid of the project name here?

Reply:
this is naming the pipeline "project.Name-pipepiper". If we don't prefix "pipepiper" with some unique id then it's possible that an old test run that failed will cause the integ test to fail